app.js

```diff
@@ -147,22 +153,27 @@ function createApp(config) {
 
   // error handler
   // eslint-disable-next-line no-unused-vars
-  app.use((err, req, res, next) => {
+  app.use((error, request, response, next) => {
+    if (!(request && request.url && request.url.includes('robots933456.txt')))
```

Comment:
I would add in a comment in the code directly here to explain this in case the azure feedback url mentioned ever changes e.g. 
// With Web App for Containers, Azure uses a special URL "/robots933456.txt" to check when the container has started, and expects a valid HTTP response (404 is fine).